package.json

```diff
@@ -4,7 +4,7 @@
   "description": "NodeJS Photo Gallery using directory structure & exif info to output a gallery",
   "version": "1.0.3",
   "homepage": "http://www.github.com/cianclarke/node-gallery",
-  l"license" : "MIT",
+  "license" : "MIT",
```

Comment:
Bit of a derp-age there OK! Just pushed a fix to master.  
It still `npm install`'s OK thought right?